src/Http/Controllers/VoyagerBaseController.php

```diff
@@ -868,7 +870,7 @@ public function relation(Request $request)
                 // If search query, use LIKE to filter results depending on field label
                 if ($search) {
                     // If we are using additional_attribute as label
-                    if (in_array($options->label, $model->additional_attributes ?? [])) {
```

Comment:
I don't understand, you just moved this.
How is it changing anything?

Reply:
The scope is applied in between his changes.
$model changes it's type then

Reply:
When you apply local scope by this `$model = $model->{$options->scope}();` then `$model` does not contains model but Builder instance which then does not support `additional_attributes`.

Try to create relation in BREAD with local scope. When you try to select it, it works but when you try to search/filter, it doesn't work. See my MR description, there is original message which is thrown